app/javascript/components/ansible-repository-form/index.jsx

```diff
@@ -71,7 +71,7 @@ const AnsibleRepositoryForm = ({ repositoryId }) => {
             message = `${__('Unable to edit Repository ')} "${values.name}". ${response.message}`;
             miqRedirectBack(message, 'warning', url);
           } else {
-            message = __(`Edit of Repository "${values.name}" was successfully initiated.`);
+            message = sprintf(__(`Edit of Repository "%s" was successfully initiated.`), values.name);
```

Comment:
I could be wrong, but instead of sprintf prefer:

```suggestion
            message = __(`Edit of Repository "%{repository}" was successfully initiated.`, values.name);
```

Reply:
Bump, is this resolved?

Reply:
@GilbertCherrie you're likely not seeing the translations because since it was changed to use `sprintf` this line needs to be retranslated.

and to answer Joe, I'll need to defer on this being solved until @Fryguy gives us his take on a named replacement helper method.

Reply:
@jrafanie we're gonna leave this as is since creating a method to allow for named replacements would be outside of the scope of this PR